posthog/test/test_feature_flags.py

```diff
@@ -1648,7 +1649,9 @@ def test_boolean_feature_flag_payload_decide(self, patch_decide):
             ),
             300,
         )
-        self.assertEqual(patch_decide.call_count, 2)
+        self.assertEqual(patch_decide.call_count, 3)
+        self.assertEqual(patch_capture.call_count, 1)
```

Comment:
testing that we call `capture` when calling `decide` with `match_value` as `None`.